### DISCUSSION TITLE
Double click to open field options slide out for field layout designer

### DISCUSSION THREAD

**Discussion by simonkuran** [craftcms/cms#11277] [Ideas] [answered]
I think it would be a great UI enhancement if double clicking on a field in the field layout designer would open the field options slide out. This would be a shortcut for clicking on the small gear icon. You can already do this to edit related elements, so my instinct is to double click on related fields too.

**Comment by brandonkelly** [comment, marked as answer]
Good idea! Added for Craft 4.1 (db8a062a95df7b68249beeed5b63af44c19976f3).

**Reply by simonkuran** [reply]
Awesome, thanks!

**Reply by brandonkelly** [reply]
Craft 4.1 is out with this change ✨
